content/collections/experiment/en/notifications.md

```diff
@@ -0,0 +1,66 @@
+this_article_will_help_you:
+  - 'Understand the type of notifications you can set for experiments'
+  - 'Create notification alerts for your experiments'
+landing: false
+exclude_from_sitemap: false
+updated_by: 3f7c2286-b7da-4443-a04f-7c225af40254
+updated_at: 1753390854
+---
+You can set up notification alerts for your experiments. Amplitude sends these notifications when your experiments transition status, when they start or end, or when major changes occur. There are two ways to receive notifications:
+
+- Email / Slack direct messages
+- Slack channel notifications / webhooks
+
+This page discusses setting up and maintaining notification alerts through Slack channels and webhooks. For information on setting up email or Slack direct message alerts, go to [Account Settings Notifications](docs/admin/account-management/account-settings#notifications) or [Integrate Slack](/docs/analytics/integrate-slack) to know how to manage your Slack workspace integration. 
+
+{{partial:admonition type="note" heading=""}}
+You must have [Manager permissions](/docs/admin/account-management/user-roles-permissions) to set up any notification alerts. However, you only need Viewer permissions to receive notifications. Talk to your Admin if you need different permissions.
+{{/partial:admonition}}
+
+Specify if you want to receive notifications for projects, deployments, or only for specific tags to a specific Slack channel or webhook.
+
+When setting notifications through webhook, you must provide the URL and valid [signing key](https://docs.knock.app/developer-tools/outbound-webhooks/overview#verifying-the-signature).
+
+##### To connect a Slack workspace to your experiment notifications
+
+You only need to connect your Slack workspace to your experiment notifications once.
+
+1. Go to *Experiment > Experiments*. 
+2. Click **Alerts** in the top right.
+3. Click **Connect To Slack**.
+4. Click **Allow** to confirm that you want to connect Amplitude to Slack.
+
+## Experiment activities that generate alerts
+You can specify if you want to receive alerts for all, or some, of the following experiment activities:
+
+- Flags created, updated, or deleted
+- A specific flag status is achieved
+- Deployments created or deleted
+
+## Setting up an alert
+
+After you have connected your Slack organization to. Amplitude, you can create notifications for your experiments. 
+
+{{partial:admonition type="note" heading=""}}
+If you want to send notifications through webhooks, you can specify your webhook at the same time that you create the alert.
+{{/partial:admonition}}
+
+##### To set up an alert
+
+1. Go to *Experiment > Experiments*. 
+2. Click **Alerts** in the top right.
+3. Specify the project that you want.
+4. Set the scope. You can select one of:
+    - **All In the Project**: Receive notifications for all experiments in the project.
+    - **By Deployment**: Receive notifications for all experiments in your entire deployment.
```

Comment:
@bgiori https://amplitude.slack.com/archives/C08TM9JDL2J/p1753387124549419 Anything about project API key we need to add more documentation on or link out to https://amplitude.com/docs/feature-experiment/data-model#deployments